tests/internal/crashtracker/test_crashtracker.py

```diff
@@ -579,3 +583,23 @@ def test_crashtracker_user_tags_core():
     for k, v in tags.items():
         assert k.encode() in data
         assert v.encode() in data
+
+
+@pytest.mark.skipif(not sys.platform.startswith("linux"), reason="Linux only")
+@pytest.mark.subprocess(env={"DD_CRASHTRACKING_ENABLED": "false"})
+def test_crashtracker_disabled_noload():
```

Comment:
this test is failing, because it explicitly loads the extension, but the asserts are to test that it doesn't get loaded.

is this test correct / what we want?

I assume we should import the core API for crash tracking instead?